docs/tutorial_basics.rst

```diff
@@ -0,0 +1,203 @@
+.. _tutorial_basics:
+
+Building a Simple Bot
+=====================
+
+
+.. note:: 
+   In this tutorial we will build a demo app with a rule-based policy. 
```

Comment:
is this still true? looks like this is learning from stories

Reply:
nope - good point :+1: